frontend/src/metabase-types/api/card.ts

```diff
@@ -155,13 +156,7 @@ export type VisualizationSettings = {
   "graph.show_stack_values"?: StackValuesDisplay;
   "graph.max_categories_enabled"?: boolean;
   "graph.max_categories"?: number;
-  "graph.other_category_aggregation_fn"?:
-    | "sum"
-    | "avg"
-    | "min"
-    | "max"
-    | "stddev"
-    | "median";
+  "graph.other_category_aggregation_fn"?: AggregationType;
```

Comment:
`DatasetColumn` uses `AggregationType` and `getDefault` uses the value from `DatasetColumn`, so I figured it would make sense

Reply:
@kulyk could you help us with this one? I'm not totally sure which is the right type. I see `other_category_aggregation_fn` used with `AggregationType` in other functions in `frontend/src/metabase/visualizations/echarts/cartesian/model/other-series.ts`
- `getOtherSeriesAggregationLabel` with `AGGREGATION_FN_MAP` 
- `getAggregatedOtherSeriesValue`

Reply:
@oisincoveney sorry for the confusion, the list of values in `AGGREGATION_FN_OPTIONS` is the source of truth here